src/treeffuser/_base_tabular_diffusion.py

```diff
@@ -98,13 +92,11 @@ def get_new_score_model(self) -> ScoreModel:
 
     def _validate_data(
         self,
-        X: Optional[ndarray] = None,
-        y: Optional[ndarray] = None,
+        X: ndarray | None = None,
```

Comment:
Nice, I didn't know this syntax.

Reply:
I learned it because `ruff` did it automatically haha.